content/swiftui/concepts/modifiers/modifiers.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: 'Modifiers'
+Description: 'Modifiers are methods that can alter views.'
+Subjects:
+ - 'Mobile Development'
+ - 'Computer Science'
+Tags:
+ - 'SwiftUI'
+ - 'SwiftUI Modifiers'
+CatalogContent:
+ - 'learn-swift'
+ - 'paths/build-ios-apps-with-swiftui'
+---
+
+In [SwiftUI](https://www.codecademy.com/resources/docs/swiftui), **Modifiers** are methods that can be called on a particular [View](https://www.codecademy.com/resources/docs/swiftui/views).
+
+## Syntax
+
+A `modifier` is a method that when called on a `View`, returns a new, altered `View`, that will replace the original `View` created. Built in `modifiers` such as `.bold()`, come built-in with Swift and all `Views` have access to these methods. `.bold()` can be used like so:
```

Comment:
The first sentence on this line should be moved up with the text on line # 15. The other sentence should be moved below the \```pseudo code block, with the text on line # 33.